src/components/VotesBar.jsx

```diff
@@ -1,10 +1,39 @@
 import React from "react";
+import { useState } from "react";
+import { useEffect } from "react";
+import { patchArticleVotes } from "../api";
+
+const VotesBar = ({ article }) => {
+  const [votes, setVotes] = useState();
+
+  const handleClick = (newVote) => {
```

Comment:
Can you think of a way of making sure a user can only vote once? Perhaps this could be determined by something in state?